powerspikegg/computation_models/utils/src/graph.cc

```diff
@@ -0,0 +1,34 @@
+#include "graph.h"
+
+namespace computation {
+namespace utils {
+
+tensorflow::Node* FindNodeWithNameInGraph(tensorflow::Graph* graph,
+                                          std::string name) {
+    for (tensorflow::Node* node : graph->nodes()) {
+        if (node->name() == name) return node;
+    }
+
+    return NULL;
+}
+
+tensorflow::Output GenerateOutputFromNode(tensorflow::Graph* graph, std::string name) {
+    tensorflow::Node* node = FindNodeWithNameInGraph(graph, name);
+    return tensorflow::Output(node);
+}
+
+void LoadGraphFromDefinition(tensorflow::Scope& scope,
+                             tensorflow::GraphDef& gDef) {
+    tensorflow::GraphConstructorOptions options;
+    tensorflow::ConvertGraphDefToGraph(options, gDef, scope.graph());
+}
+
+void LoadGraphFromFile(tensorflow::Scope& scope, std::string filename) {
+    tensorflow::GraphDef gDef;
+    tensorflow::ReadBinaryProto(tensorflow::Env::Default(), filename, &gDef);
+
+    tensorflow::GraphConstructorOptions options;
+    tensorflow::ConvertGraphDefToGraph(options, gDef, scope.graph());
```

Comment:
Shouldn't you use void LoadGraphFromDefinition(tensorflow::Scope& scope, tensorflow::GraphDef& gDef) ?